finetune/base_models/bert/featurizer.py

```diff
@@ -49,7 +52,7 @@ def bert_featurizer(
         hidden_act=config.act_fn,
         hidden_dropout_prob=config.resid_p_drop,
         attention_probs_dropout_prob=config.attn_p_drop,
-        max_position_embeddings=config.max_length, 
+        max_position_embeddings=max_length, 
```

Comment:
Wild -- there are BERT position embeddings up to 2048 (even for small model architecture)?! Curious if they are trained or just randomly initialized / never used during pre-training.

Reply:
Not sure haha, didn't think about this